galaxy/main/models/collection.py

```diff
@@ -68,6 +69,13 @@ def latest_version(self):
             collection=self
         ).latest('pk')
 
+    @property
+    def highest_version(self):
+        versions = CollectionVersion.objects.filter(collection=self)
+        pairs = [(v, semver.Version(v.version)) for v in versions]
```

Comment:
`return sorted(versions, key=lambda x: semver.Version(x.version))[0]`

Reply:
Could potentially provide a `semantic_version.Spec()` here and use it's `select()` method: https://python-semanticversion.readthedocs.io/en/latest/reference.html#semantic_version.Spec.select

For now, the `Spec()` used could just be '*', but using the Spec would offer a way to support other version restrictions.

Reply:
`Spec()` was added as the selector